### ISSUE TITLE
Add example png to readme

### ISSUE THREAD

**Issue by jhermann** [minimaxir/stylistic-word-clouds#1]
Show us how it looks like.

**Comment by rodw** [comment]
Agreed. For what it is worth the `wordclouds` subdirectory contains what I assume are examples of wordclouds generated with this library.

The other obvious thing missing from the README is some sort of usage example, but based on a very cursory look at the *.py files I think there is no generic command-line interface to this, I believe a small custom py script generates each image.
